thrall/app/lib/Paging.scala

```diff
@@ -0,0 +1,22 @@
+package lib
+
+import play.api.mvc.{Result, Results}
+
+import scala.concurrent.Future
+
+case class Paging(page: Int, from: Int, pageSize: Int)
+
+object Paging extends Results {
+  def withPaging(maybePage: Option[Int])(f: Paging => Future[Result]): Future[Result] = {
+    val pageSize = 250
```

Comment:
Wonder if it's worth noting any reason that this number is chosen in particular.

Reply:
Would be a good idea - but there's no particular reason for this number, and no barrier to changing in the future!